paf-mvp-client-express/src/client-node.ts

```diff
@@ -45,7 +47,17 @@ export class ClientNode extends Node {
     const hostName = config.host;
     const operatorHost = config.operatorHost;
 
-    this.client = new OperatorClient(operatorHost, hostName, currentPrivateKey, this.publicKeyProvider);
+    const dsaBuilder = new ECDSA_NIT_P256Builder();
```

Comment:
I recommend passing a builder in the `OperatorClient` constructor instead: at some point it will probably need a `verifier` as well, and it already has the private key as parameter.

Reply:
Make sense ok.